cmd/pulumi-test-language/README.md

```diff
@@ -25,11 +25,29 @@ For each test run and language, then:
   separate process and the assertions are checked, exercising program execution
   for the given language.
 * Generated code is snapshot tested to ensure that it doesn't change
-  unexpectedly.[^pulumi-accept]
+  unexpectedly.
 
-[^pulumi-accept]:
-    Snapshots can typically be updated by running tests with the `PULUMI_ACCEPT`
-    environment set to a truthy value (e.g. `PULUMI_ACCEPT=1 go test ...`).
+## Running Language Conformance Tests
+
+To run the language conformance tests, for example for Python, run:
+
+```bash
+go test github.com/pulumi/pulumi/sdk/python/cmd/pulumi-language-python/v3 -count 1
+```
+
+Note: to run this from the root of the repository, make sure you have an uptodate `go.work` file by running `make work`.
+
+To run a single test case:
+
+```bash
+go test github.com/pulumi/pulumi/sdk/python/cmd/pulumi-language-python/v3 -count 1 -run TestLanguage/classes/l2-map-key
```

Comment:
Is it worth mentioning the difference between languages here?  The exact name is dependent on which types of tests each language runs iirc?

Reply:
Separately, it occurs to me that we should add a `Makefile` target (or set thereof) for doing this, if we don't have them already.

Reply:
Yeah, looking at it and Python runs tests in 3 variants:

* setuppy
* toml
* classes

So you get `TestLanguage/${variant}/${test_name}`

Nodejs runs them with 2 variants

* forceTsc=true
* forceTsc=true

TestLanguage/forceTsc=true/l

Reply:
Update with info on the variant, let me know if it makes sense.